cpp/tests/data/Dataset.cpp

```diff
@@ -330,6 +335,38 @@ TEST(Dataset, PLYPointCloud) {
     utility::filesystem::DeleteDirectory(extract_dir);
 }
 
+TEST(Dataset, PTSPointCloud) {
+    const std::string prefix = "PTSPointCloud";
+    const std::string data_root = utility::filesystem::GetHomeDirectory() +
```

Comment:
Future work: use https://en.cppreference.com/w/cpp/filesystem/temp_directory_path

Reply:
Actually, what about something different: don't create an explicit test directory, but allow local unit tests to have the files cached **If the downloaded file belongs to one of the standard datasets**. 

- On the local machine: cached download files can make the unit test run faster.
- On the remote machine: CI will always start fresh. The download will always be tested.